blog/2020-10-29-macos-arm64.md

```diff
@@ -107,7 +107,7 @@ bootstrapper.
 
 All the below changes will be done by a bot and the packages the bot
 will send PRs to is determined by the list of packages at
-[conda-forge-pinning](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/migrations/osx_arm64.txt)
+[conda-forge-pinning](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/main/recipe/migration_support/osx_arm64.txt)
```

Comment:
I didn't go digging in old blog posts and minutes, which I think are probably better left alone as a historical record. The risk that anyone is reading blog posts from 5y ago is relatively low too. 😅

Reply:
The link checker fails. So I changed it.

Reply:
OK, makes sense, thx